dev

```diff
@@ -342,21 +365,45 @@ boot() {
     append_args_file
     APPS="$(apps_to_load)"
 
-    BOOT_SEQUENCE="
-        Apps=[${APPS}],
-        ok=lists:foreach(fun application:load/1, Apps),
-        io:format(user, \"~nLoaded: ~p~n\", [Apps]),
-        {ok, _} = application:ensure_all_started(emqx_machine).
-    "
+    if [ "${IS_ELIXIR:-}" = "yes" ]; then
+        BOOT_SEQUENCE='
+          apps = System.get_env("APPS") |> String.split(",") |> Enum.map(&String.to_atom/1)
```

Comment:
Nit: Couldn't we move the boot sequence somewhere into Elixir/Erlang code 🤔 

Also, maybe much Bash boilerplate may be moved to `mix.exs` script in Elixir case to avoid doing it in Bash and making Bash <-> Elixir exchange 🤔

Reply:
I'm trying to piggyback on top of the existing script, that's why I didn't touch `mix.exs` too much.  :thinking: